feat: Add new admin command to print Argo CD initial password (11117) (#11155)
argoproj/argo-cd: docs/user-guide/commands/argocd_admin_initial-password.md

## argocd admin initial-password

> Prints initial password to login into Argo CD for the first time

Reviewer: ```suggestion
Prints initial password to log in to Argo CD for the first time
```
